### ISSUE TITLE
Social Settings are not displayed correctly

### ISSUE THREAD

**Issue by gadgethm** [andrewbanchich/dimension-jekyll-theme#1]
When using the contact page with this theme, the social settings display as raw text.  It looks like the liquid tags are not being processed.  Anyone else see this issue?  Did I forget to install something?
![image](https://user-images.githubusercontent.com/10714864/35947958-7c47c720-0c27-11e8-9772-729bdc3e2eeb.png)

Thanks!

**Comment by andrewbanchich** [comment]
How are you running this? More info would help.

**Comment by gadgethm** [comment]
My current setup is to have Jekyll run locally, then upload the site to github pages.  You can view the actual repo at https://github.com/gadgethm/gadgethm.github.io, which contains both the source and the jekyll-built result.  I modified the contact.md file to remove the 'layout: default' line.  This enables it to appear as a button on the main page.  Clicking that button results in the screen above.

Let me know if you need more details, I'm not sure if you're looking for something specific.
Thanks for the help!

**Comment by gadgethm** [comment]
So my problem has been solved.  It appears that using jekyll serve to serve a local version of the site does not process all pages' liquid tags.  However, when calling jekyll build instead, the site builds correctly and all links seem to work.  Since I was running on a test environment, it seemed like my website was not working right, but after publishing some changes it looks okay now. 

I'll close this issue for now, and look into why 'jekyll build' works but 'jekyll serve' does not.

**Comment by andrewbanchich** [comment]
That's still odd. `jekyll serve` should process liquid tags since the whole point is to view the site before building it. I'd guess it is something about your local setup, though, since it works on GitHub Pages. What was your install process for Jekyll? Does it show any errors in the console when running `jekyll serve`?

**Comment by andrewbanchich** [comment]
Also, it looks like you're publishing the fully built Jekyll site?

**Comment by gadgethm** [comment]
I followed the Jekyll tutorial to install for github pages, but inside the source branch (After cloning your repo, used Gem commands to install jekyll and the dimension theme). My Gem update process is to call bundle install, followed by bundle update.  'bundle update' results in one error with ffi, but it doesn't seem related, as the older version of ffi still works.  

When running 'bundle exec jekyll serve', I get a warning from my rakefile, but nothing else and no errors from jekyll itself.  

Yes, my publishing strategy is to use Github only to publish the static site, as Github has issues with third party themes and gems.  So to get around that, I run jekyll locally to build, then use the resulting site directly in Github's master branch (using a .nojekyll file to disable Github's automatic run of jekyll).

Also, it appears even the 'jekyll serve' version is working now and I have no idea what I changed.  If the issue comes back, I'll try to check jekyll's log for more details.  This is very strange.
